dev/archery/archery/lang/rust.py

```diff
@@ -1,23 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I am not sure this one should be removed @jorgecarleitao  -- it might be how rust is run in the integration test framework.